src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -157,6 +157,7 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp env', arg_group='Dapr') as c:
         c.argument('instrumentation_key', options_list=['--dapr-instrumentation-key'], help='Application Insights instrumentation key used by Dapr to export Service to Service communication telemetry')
+        c.argument('connection_string', options_list=['--dapr-connection-string', '-d'], help='Application Insights connection string used by Dapr to export Service to Service communication telemetry.')
```

Comment:
can we just use dapr_connection_string, since connection string is much generic, if we add some other connection string properties, it might be confused.